parl/framework/model_base.py

```diff
@@ -124,23 +124,39 @@ class Model(Network):
     In conclusion, Model is responsible for forward and 
     Algorithm is responsible for backward.
 
-    Model can also be used to construct target model, which has the same structure as initial model.
+    Model can also use deepcopy way to construct target model, which has the same structure as initial model. 
+    Note that parameters in the initial model havn't been copied to target model.
```

Comment:
Note that only the model definition are copied here. To copy the parameters from the current model to the target model, you must explicitly use `sync_params_to`  function after the program is initialized.

Reply:
done